web_poet/page_inputs/client.py

```diff
@@ -22,6 +25,18 @@
 _StatusList = Union[str, int, List[Union[str, int]]]
 
 
+@dataclass
+class SavedResponseData:
```

Comment:
I made this a real class when I still thought we will need to add an exception field here, now we can make it a simple tuple again but I like named fields.

Reply:
Would it make sense to call it _SavedResponseData?

Reply:
We import it in other parts of web-poet, though I don't think other projects would need to import it, so I think yes?

Reply:
I see. Yes, it seems making it private is better, as it's not something users should use.